cmd/block-explorer/main.go

```diff
@@ -13,16 +13,16 @@ import (
 	"syscall"
 
 	"github.com/insolar/block-explorer/api"
-	"github.com/insolar/insconfig"
-	"github.com/insolar/insolar/ledger/heavy/exporter"
-	"github.com/pkg/errors"
-
 	"github.com/insolar/block-explorer/etl/connection"
 	"github.com/insolar/block-explorer/etl/controller"
+	"github.com/insolar/block-explorer/etl/dbconn/plugins"
 	"github.com/insolar/block-explorer/etl/extractor"
 	"github.com/insolar/block-explorer/etl/processor"
 	"github.com/insolar/block-explorer/etl/transformer"
 	"github.com/insolar/block-explorer/instrumentation/belogger"
+	"github.com/insolar/insconfig"
+	"github.com/insolar/insolar/ledger/heavy/exporter"
+	"github.com/pkg/errors"
```

Comment:
Can you group current project imports please? It sets in Golang Preferences->Editor->Code Style->Go